terraform/workspace_variables/production.tfvars.json

```diff
@@ -9,9 +9,19 @@
   "postgres_database_service_plan": "small-ha-13",
   "hostnames": ["find-a-lost-trn"],
   "statuscake_alerts": {
-    "tra-flt-prod": {
+    "tra-flt-prod-1": {
       "website_name": "find-a-lost-trn-production",
-      "website_url": "https://find-a-lost-trn-production.london.cloudapps.digital/health",
+      "website_url": "https://find-a-lost-trn-production.london.cloudapps.digital/health/all",
```

Comment:
There should only be one check for the `find-a-lost-trn.education.gov.uk` domain; I don't think we need to test the cloudapps.digital one, as it's the same app, just a different CNAME.

Reply:
It's interesting to keep both to identify if the problem is with paas or cloudfront (london.cloudapps.digital bypasses cloudfront)

Reply:
@saliceti ah ok, makes sense